invent-code-0/main.cpp

```diff
@@ -4,4 +4,8 @@ using namespace std;
 
 int main() {
   // Write code here.
+  cout << "My name is Cyndi,\n";
```

Comment:
Can you replace the "," with "."? Thanks!

Reply:
Done.